.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.inlayHints.parameterNames.enabled": "all",
   "typescript.inlayHints.parameterTypes.enabled": true,
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "cSpell.words": ["Ayurved", "Banousadhi", "Bhandar", "tailwindcss"]
```

Comment:
Nice!
Do you have the cSpell extension enabled?

Reply:
no, I had added these words to the add to vscode workspace and this line got added automatically

Reply:
cSpell is a VS Code extension to find typos. This line will tell cSpell that those words should be considered as correct.